.travis.yml

```diff
@@ -1,19 +1,24 @@
 language: python
 python:
-- '3.5'
 - '3.6'
 jdk:
   - openjdk8 # recommended jdk for solr < 7
 env:
   global:
-    - SOLR_VERSION=6.6.5
     - JAVA_HOME=/usr/lib/jvm/java-1.8.0-openjdk-amd64  # otherwise jdk11 will be used
   jobs:
-    - DJANGO=
-    - DJANGO=1.11
-    - DJANGO=2.0
-    - DJANGO=2.1
-    - DJANGO=2.2
+    # solr 8
+    - SOLR_VERSION=8.6.2 DJANGO=
+    - SOLR_VERSION=8.6.2 DJANGO=1.11
+    - SOLR_VERSION=8.6.2 DJANGO=2.0
+    - SOLR_VERSION=8.6.2 DJANGO=2.1
+    - SOLR_VERSION=8.6.2 DJANGO=2.2
```

Comment:
mep and ppa are both on django 2.2 now, and versions prior to 2.2 are no longer supported by django — let's go ahead and drop them. (We can do that in a separate PR + chore if it makes more sense to you!)